proposals/collection-literals.md

```diff
@@ -451,7 +439,7 @@ Not having a *known length* does not prevent any result from being created. Howe
     *  If `T` is some `Span<T1>`, then the literal is translated as the same as above, except that the `__result` initialization is translated as:
 
         ```c#
-        Span<T1> __result = stackalloc T1[__len];
```

Comment:
What does the absence of `new` or `stackalloc` mean? Is this like saying `new`/`stackalloc`?

Reply:
Thanks, this should be `new T1[__len]`.

Reply:
just to be clear though: the impl should be free to stack-alloc this space if it feels like it is appropriate.  LIkely using the same heuristic that `params Span<T>` would use to determine if the space should be on the stack or the heap.  I don't mind the translation saying it's an array, as long as we make it clear it allowed to *not* be an array.

Reply:
Add a comment about using `stackalloc`.